aroma/aroma.py

```diff
@@ -35,8 +32,35 @@ def aroma_workflow(
 
     Parameters
     ----------
-    in_feat
+    in_file : str
+        Path to MNI-space functional run to denoise.
+    mc : str
+        Path to motion parameters.
+    mixing : str
+        Path to mixing matrix.
+    component_maps : str
+        Path to thresholded z-statistic component maps.
+    mask : str
+        Path to binary brain mask, in MNI space.
```

Comment:
This could probably be made optional with a default to a standard MNI brain mask, but I didn't want to make any enhancements like that yet.